freezefrog/__init__.py

```diff
@@ -1,6 +1,10 @@
 import calendar
 import datetime
-from mock import patch
+
+try:
+    from mock import patch
+except ImportError:
+    from unittest.mock import patch
```

Comment:
Should this go the other way round? I.e. give priority to the built-in?

Reply:
Sure